modules/ROOT/pages/cryptography-jce.adoc

```diff
@@ -658,3 +658,10 @@ Validate a stream against a signature, using a key.
 | Alias a| String | Alias of the key in the keystore. |  | x
 | Password a| String | Password used to unlock the key. |  | x
 |===
+
+== JCE PBE Encrypt and Decrypt
+
+When using the JCE PBE Encrypt or Decrypt operations, consider the following specifications:
+
+* The length of the initialization vector (IV) is the cipher block size returned when calling `javax.crypto.Cipher.getBlockSize()`. For example, if you use AES 128, then the block size is 16 bytes, so the IV has a length of 16 bytes.
+* The key is generated from the password with 0 iterations and a null salt value for encrypt and decrypt. For signing and validation, the Mac is generated with 20 iterations and a fixed salt value hardcoded in the connector's code.
```

Comment:
```suggestion
* The operation generates the key from the password with 0 iterations and a null salt value for encrypt and decrypt. For signing and validation, the operation generates the Mac key with 20 iterations and a fixed salt value hardcoded in the connector's code.
```
Suggestion to avoid passive voice. Also added "key" after "the Mac", please confirm if this is accurate.